docker/mu-plugins/conflicting-plugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * Plugin Name: Conflicting Plugin
+ * License: GPL2+
+ */
+error_log( 'loading conflicting plugin' );
+/**
+ * THIS CODE WOULD NEED TO BE DUPLICATED IN EACH PLUGIN...
+ */
+if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
+	function jetpack_enqueue_library( $class_name, $version, $path ) {
+		global $jetpack_libraries;
+		if ( ! is_array( $jetpack_libraries ) ) {
```

Comment:
You don't have to make this check every time if you just add the initialization outside / above this functiion.

```php
if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
  global $jetpack_libraries;
  if ( ! is_array( $jetpack_libraries ) ) {
    $jetpack_libraries = array();
  }

  function jetpack_enqueue_library( $class_name, $version, $path ) {
    global $jetpack_libraries;
    if ( ! isset( $jetpack_libraries[ $class_name ] 
      || version_compare( $jetpack_libraries[ $class_name ] ['version'], $version, '>' )
      ) ) {
        $jetpack_libraries[ $class_name ] = array( 'version' => $version, 'path' => $path );
    }
  }
  // ...
}
```